### PR TITLE
Bpfctl use comfy table for all commands

### PR DESCRIPTION
**Pull request by astoycos**
Pulled out of #656

### CODE REVIEW

**Comment by astoycos** [comment]
image

File: bpfctl/src/main.rs [hunk 1]
```diff
@@ -321,15 +325,212 @@ struct GlobalArg {
...
+        let mut table = Table::new();
+
+        table.load_preset(comfy_table::presets::NOTHING);
+        table.set_header(vec![Cell::new("Kernel State")
+            .add_attribute(comfy_table::Attribute::Bold)
+            .add_attribute(comfy_table::Attribute::Underlined)
+            .fg(Color::Green)]);
+
+        let rows = vec![
+            vec!["Kernel ID:".to_string(), r.btf_id.to_string()],
```
[the remark this answers is not in the document]
Comment (astoycos) [reply bpfctl/src/main.rs:488]: Good catch... Also Bpfd State should just return NONE not all that 👍

**Comment by astoycos** [comment]
Fixed up 

image

**Comment by Billy99** [comment]
I'm confused on what the `BTF ID` is. Using your output above, the `BTF ID` and the `Kernel ID` are the same (5394), but different from the Kernel/Program Id of 8123 used to retrieve the data. I thought the 8123 was the Kernel ID?

**Comment by astoycos** [comment]
I messed up again @Billy99 Sorry moving too fast

File: bpfctl/src/main.rs [hunk 2]
```diff
@@ -320,15 +324,217 @@ struct GlobalArg {
...
```
Comment (astoycos) [inline bpfctl/src/main.rs:492]:
```suggestion
            vec!["Kernel ID:".to_string(), r.bpf_id.to_string()],
```

**Comment by astoycos** [comment]
Fixed 

image

**Review by Billy99 — APPROVED** [review]
LGTM

**Pull request merged**